### DISCUSSION TITLE
Health Scale in Minestom?

### DISCUSSION THREAD

**Discussion by Oliton** [Minestom/Minestom#2133] [Help] [answered]
Is there a way I can compress heart amounts using a scale or something? (If not, I assume I gotta use a custom health system). 
I just want to set it so that if the player's heath is 100 the hearts show a percentage of that (keeping the regular 10 hearts).

**Comment by FrederikHeinrich** [comment, marked as answer]
you have to do a custom health system for that. :)

**Reply by Oliton** [reply]
ok cool thanks :)
